app/workers/backend_delete_service_worker.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class BackendDeleteServiceWorker
+  include Sidekiq::Worker
+
+  def self.enqueue(event)
+    perform_async(event.event_id)
+  end
+
+  def perform(event_id)
+    event = EventStore::Repository.find_event!(event_id)
```

Comment:
If the event does not exist, it will raise an error and Sidekiq will schedule a retry. If the event does not exist, what's the point in retrying?

Reply:
@guicassolato Right, it shouldn't retry, but we should inform to bugsnag because that would mean that we have a bug if that happens... I will do that 😉 👍

Reply:
Done 😄

Reply:
Sorry, I don't see what changed.

Reply:
@guicassolato line 20:
```ruby
rescue ActiveRecord::RecordNotFound => exception
    System::ErrorReporting.report_error(exception, parameters: {event_id: event_id})
```

Reply:
Oops. My bad. 👍